### PR TITLE
KAZOO-3447: When port is scheduled send an email

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

File: applications/crossbar/priv/couchdb/schemas/port_requests.json [hunk 1]
```diff
@@ -1,133 +1,140 @@
...
         },
-        "extended_address": {
-          "type": "string",
-          "required": false,
-          "name": "Address",
-          "description": "The suit/floor/apt of the billing address the loosing carrier has on record"
+        "scheduled_date": {
+            "type": "integer",
+            "required": false,
+            "name": "Transfer Date",
```
Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: Shouldn't name be "Scheduled Date"?

File: applications/crossbar/priv/couchdb/schemas/port_requests.json [hunk 2]
```diff
@@ -1,133 +1,140 @@
...
+                    "type": "string",
+                    "required": false,
+                    "name": "Address",
+                    "description": "The address of the billing address the loosing carrier has on record"
+                },
+                "extended_address": {
+                    "type": "string",
+                    "required": false,
+                    "name": "Address",
+                    "description": "The suit/floor/apt of the billing address the loosing carrier has on record"
```
Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: s/suit/suite

Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: s/loosing/losing

File: applications/crossbar/priv/couchdb/schemas/port_requests.json (hunk 1 above)
Comment (macpie) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: oh copy/paste !

File: applications/crossbar/priv/couchdb/schemas/port_requests.json (hunk 2 above)
Comment (macpie) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: ?

File: applications/crossbar/src/modules/cb_port_requests.erl [hunk 3]
```diff
@@ -1127,6 +1139,20 @@ send_ported_notification(Context, Id) ->
...
 %% @end
 %%--------------------------------------------------------------------
+-spec send_port_scheduled_notification(cb_context:context(), ne_binary()) -> 'ok'.
+send_port_scheduled_notification(Context, Id) ->
+    Req = [{<<"Account-ID">>, cb_context:account_id(Context)}
+           ,{<<"Authorized-By">>, cb_context:auth_account_id(Context)}
+           ,{<<"Port-Request-ID">>, Id}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    whapps_util:amqp_pool_send(Req, fun wapi_notifications:publish_port_scheduled/1).
```
Comment (jamesaimonetti) [inline applications/crossbar/src/modules/cb_port_requests.erl]: This won't wait for a response from teletype for whether the email was sent or not, will always return 'ok', and the API call will succeed. Seems...not right.

File: applications/crossbar/priv/couchdb/schemas/port_requests.json (hunk 2 above)
Comment (macpie) [inline applications/crossbar/priv/couchdb/schemas/port_requests.json]: vim users ...

**Comment by jamesaimonetti** [comment]
Manually merged because I rebased master back in.

**Pull request closed without merging**